internal/export/distro/registrations_test.go

```diff
@@ -9,15 +9,16 @@ package distro
 import (
 	"testing"
 
+	"github.com/edgexfoundry/edgex-go/internal"
 	"github.com/edgexfoundry/edgex-go/internal/export"
+
+	"github.com/edgexfoundry/edgex-go/pkg/clients/logging"
 	"github.com/edgexfoundry/edgex-go/pkg/models"
-	"go.uber.org/zap"
 )
 
 func init() {
-	if logger == nil {
-		logger = zap.NewExample()
-		_ = logger
+	if LoggingClient == nil {
```

Comment:
My preference would be to setup these kinds of dependencies in a TestMain function. See internal/core/data/event_test.go